Source/Csla/DataPortalClient/HttpProxy.cs

```diff
@@ -82,7 +85,15 @@ HttpClientHandler CreateDefaultHandler()
         var handler = new HttpClientHandler();
         if (!Options.UseTextSerialization)
         {
-          handler.AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
+#if NET8_0_OR_GREATER
```

Comment:
Instead of this double `#if` wouldn't it be easier to do something like this?
```csharp
bool browserOS = false;
#if NET8_0_OR_GREATER
browserOS = OperatingSystem.IsBrowser();
#endif
if(!browserOS && !Options.UseTextSerialization)
{
 // ...
}
```

Reply:
Honestly I try to avoid any variable creation (maybe that's overkill), but can you take a look at other variant which I come up with.

Reply:
I'm not against that. I personally prefer multiple return statements in a method to shortcut the method.
But csla tries to avoid multiple statements as far as I know. Maybe @rockfordlhotka  can shed some light/preference here.

Reply:
I do prefer to have a single `return` statement in a method. However, if there's a compelling reason (code clarity, etc.) for multiple statements I'm ok with it as an exception.